### PR TITLE
Fix(Core/Spell): No visual for Suppression Aura

### PR DESCRIPTION
**Pull request by temperrr**
## Changes Proposed:
-  Changes visual to 0.
-  

## Issues Addressed:

- Partially closes https://github.com/azerothcore/azerothcore-wotlk/issues/11275

## Tests Performed:

-  Tested
- 

## How to Test the Changes:

1. Go to suppression room in bwl
2. check if the fire nova effect is gone but the mist affect stays
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: src/server/game/Spells/SpellInfoCorrections.cpp [hunk 1]
```diff
@@ -4167,10 +4167,12 @@ void SpellMgr::LoadSpellInfoCorrections()
         spellInfo->Effects[EFFECT_0].RadiusEntry = sSpellRadiusStore.LookupEntry(41); // 150 yards
         spellInfo->AttributesEx2 |= SPELL_ATTR2_IGNORE_LINE_OF_SIGHT;
     });
-
+    
+    // Suppression Room Aura (BWL)
```
Comment (Kitzunu) [inline src/server/game/Spells/SpellInfoCorrections.cpp:4171]:
```suggestion
    // Suppression Aura
```

**Review by Kitzunu — CHANGES_REQUESTED** [review]

**Comment by Lynethris** [comment]
#11337 is it not fix this ?

**Comment by pklloveyou** [comment]
> @temperrr #11337 is it not fix this ?

No, Spell id13813 is also wrong

**Comment by pklloveyou** [comment]
There may be many of the same mistakes

**Comment by Lynethris** [comment]
@UltraNix

**Comment by UltraNix** [comment]
@Lynethris ??

**Comment by Lynethris** [comment]
> @Lynethris ??

@pklloveyou speak about some errors in your fixs #11341 and #11337. Spells ID will be wrong and this fix requiered. I didn't understand everything. For me, we have a duplicate fix. I'm wrong ?

**Comment by Lynethris** [comment]
> > @Lynethris ??
> 
> @pklloveyou speak about some errors in your fixs #11341 and #11337. Spells ID will be wrong and this fix requiered. I don't really understand. For me, we have a duplicate fix. Am I wrong ?

**Comment by UltraNix** [comment]
I don't understand @mpfans as well.

**Comment by Lynethris** [comment]
I have quickly check this morning on the actual master branch and #11337 have fix the visual aura from suppression device.

WoWScrnShot_041522_134335

**Pull request closed without merging**